README.md

```diff
@@ -1,71 +1,67 @@
-problems or errors it encounters along the way. For a large directory there will be a lot of output,
-but it should be easily searchable. 
-
-For a file, the tool will attempt to transform that file
-and then attempt to verify that file. No summary output is provided.
+- You are not running the tool on Windows (no path separator support, see https://github.com/Ecosystem-Infra/web-test-transformer/issues/11). 
+- You must be in `chromium/src/third_party/blink/web_tests` within the Chromium source tree (for verification)
+- You must have content_shell built. (for verification)
+Steps:
+1. Run the tool on some tests
+2. Make note of the summary output: particularly transformation and verification failures
+3. Manually review or restore failures, make adjustments on tests
+4. Commit any transformations you wish to keep.
+5. Send a CL for review and CQ run
 
 
-For a directory, the tool will iterate through the files in that directory (non-recursively), only transforming
-.html tests. Each file is handled like a single file above (transformed then verified). It is not
-batch-transformed then batch-verified.
+### Dry Run
+By default, the tool overwrites the original file and changes can be viewed using `git diff`.
+Sometimes that output can be hard to read and understand the full context.
+If you would like to write the transformed file to a different path to compare the files, 
+create a directory and pass the path to the `--output_dir` flag. The transformed files will
+appear in that directory with their original names.
 
-After running on a directory, the tool provides summary output detailing the transformations that completed 
-(does __NOT__ guarantee correctness, just completion of the transformation code), failed, and
-files that were skipped. The skipping reason is logged, for example the test does not src `js-test.js`.
-The tool also lists whether transformed files succeeded or failed verification.
-With this summary output, you can manually check files failed transformations/verifications to see if the
-problem is easily fixed by a human, report bugs with this tool if the transformation should have worked, or just 
-`git checkout` those files and commit the succesful transformations to Gerrit.
+**Dry Run Example**
 
-The tool currently only transforms .html tests. If a test src's a helper .js script, neither file will be 
-transformed and the test will be marked as skipped. If a test src's a helper .js script __and__ `js-test.js`,
-the test file will be transformed, marked as completed, but verification will fail.
+`$mkdir transformed_temp`
 
-**Example**
-`$node ~/web-test-transfomer/index.js --dir=fast/dom/SelectorAPI --target_build=Debug`
+`$node ~/web-test-transformer/index.js --dir=./fast/dom/SelectorAPI --output_dir=./transformed_temp`
 
-### Dry Run
-If you would like to do a dry run to compare the files, create a directory and pass the path
-to the `--output_dir` flag.
+*review the transformations*
```

Comment:
so continuing this example, this could be something like the following?

$diff ./fast/dom/SelectorAPI ./transformed_temp/fast/dom/SelectorAPI